src/main.c

```diff
@@ -1016,6 +1020,19 @@ int mainloop(int toplevel)
   return (eggbusy || tclbusy);
 }
 
+void init_random() {
+  unsigned int seed;
+#ifdef HAVE_GETRANDOM
+  if (getrandom(&seed, sizeof(seed), 0) != sizeof(seed))
+    fatal("ERROR: getrandom()\n", 0);
+#else
+  struct timeval tp;
```

Comment:
I think we can make the function static, as it is only used here, if you agree. functions with no arguments should have "void" as argument specification, init_random() means "unspecified arguments". `static void init_random(void)` would be my proposal.

Reply:
made it static now. thank you. but init_random() {...} is function definition, not function declaration. so it does not mean unspecified arguments in this case.